mappings/net/minecraft/data/server/recipe/CookingRecipeJsonBuilder.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_2454 net/minecraft/data/server/recipe/CookingRecipeJsonFactory
+CLASS net/minecraft/class_2454 net/minecraft/data/server/recipe/CookingRecipeJsonBuilder
 	FIELD field_11414 experience F
 	FIELD field_11415 cookingTime I
 	FIELD field_11416 builder Lnet/minecraft/class_161$class_162;
```

Comment:
Should this be `advancementBuilder`? (Same in `Shaped/Shapeless/SingleItem/SmithingRecipeJsonBuilder`)